### PR TITLE
doc: add github stats example

### PR DESCRIPTION
**Pull request by Pryz**

### CODE REVIEW

File: examples/github_stats/app.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    contributors_url = repo_info["contributors_url"]
+    response = httpx.get(contributors_url)
+    response.raise_for_status()
+    contributors = response.json()
+    return contributors
+
+
+@dispatch.coroutine()
+async def main():
+    repo_info = await get_repo_info.call("stealthrocket", "coroutine")
```
Comment (achille-roussel) [inline examples/github_stats/app.py:47]: What do you think about https://github.com/stealthrocket/dispatch-sdk-python/pull/79 and the idea of dropping `.call` in favor of a simple function call?

**Review by achille-roussel — COMMENTED** [review]
Let's update to remove the `.call` and ship it?

File: examples/github_stats/test_app.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        app.get_contributors._client = server.client
+        app.main._client = server.client
+
+        http_client = TestClient(app.app, base_url="http://dispatch-service")
+        app_client = function_service.client(http_client)
+
+        response = http_client.get("/")
+        self.assertEqual(response.status_code, 200)
+
+        server.execute(app_client)
```
Comment (pelletier) [inline examples/github_stats/test_app.py:39]: Given the example I think this should be ran multiple times to verify that multiple functions are indeed called. This is possible as of https://github.com/stealthrocket/dispatch-sdk-python/pull/76. For example: https://github.com/stealthrocket/dispatch-sdk-python/blob/main/examples/auto_retry/test_app.py#L44

File: examples/github_stats/test_app.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import unittest
+from unittest import mock
+
+from fastapi.testclient import TestClient
+
+from ... import function_service
+from ...test_client import ServerTest
+
+
+class TestAutoRetry(unittest.TestCase):
```
Comment (pelletier) [inline examples/github_stats/test_app.py:15]:
```suggestion
class TestGithubStats(unittest.TestCase):
```

File: examples/github_stats/test_app.py (hunk 2 above)
Comment (Pryz) [reply examples/github_stats/test_app.py:39]: Calling multiple execute doesn't give me a new response somehow.

Comment (pelletier) [reply examples/github_stats/test_app.py:39]: OK feel free to merge, I'll look into it. Might have missed something in #76.

**Review by pelletier — APPROVED** [review]

**Pull request merged**